ale_linters/rust/analyzer.vim

```diff
@@ -26,10 +27,28 @@ function! ale_linters#rust#analyzer#GetProjectRoot(buffer) abort
     return ''
 endfunction
 
+function! ale_linters#rust#analyzer#GetConfig(buffer) abort
+    let l:config = copy(ale#Var(a:buffer, 'rust_analyzer_config'))
+
+    if ale#Var(a:buffer, 'rust_analyzer_use_local_config')
+        let l:config_local_path = ale#path#FindNearestFile(a:buffer, 'analyzer.json')
```

Comment:
As mentione in previous command I rather have this be a filename rather than a boolean. Is more flexible in that users can set any name for the config to match their projects conventions.

Check if the file is readable and if not then we assume no configuration file is present.

Reply:
But we need a try/catch anyway, 'cause the json content might be mis-formatted, in this case to would flood the the buffer.